blogs/2022-06-28-tips-kafka-streams-distributed.md

```diff
@@ -39,17 +39,17 @@ executions
     .to("executions", Produced.with(Serdes.String(), JsonSerde.of(Execution.class)));
 ```
 
-In that case, a concurrent process can write on the same topic, overwriting the previous value and destroying the previous computed methods.
-For that usage, you must define a single writer for a Key at the same time. That led to our next part: a custom joiner
+In that case, a concurrent process can write on the same topic, overwriting the previous value and dismantling the previous computing methods.
+In such a context, you must define a single writer for a key at the same time. This leads to our next part: the custom joiner.
 
 ## Custom Joiner
-In order to process execution, we have separated microservice as multiple topics, the easier topology to understand is:
-- a topic with the executions (with multiple tasks)
-- a topic with task results.
+To process execution, we have separated microservice into multiple topics. The more straightforward topology to understand is:
```

Comment:
Again, I think here we need to set the context for the sub-header with a general statement about the concept of "custom joiner." How does it relate to the overall article?